elastic4s-http/src/main/scala/com/sksamuel/elastic4s/http/search/SearchImplicits.scala

```diff
@@ -20,7 +20,13 @@ trait SearchImplicits {
   }
 
   implicit object SearchShow extends Show[SearchDefinition] {
-    override def show(req: SearchDefinition): String = SearchBodyBuilderFn(req).string()
+    override def show(req: SearchDefinition): String = SearchHttpExecutable.body(req)
+  }
+
+  implicit object SearchPrintableRequest extends PrintableRequest[SearchDefinition] {
```

Comment:
Worth considering whether it would be better to just have `SearchHttpExecutable` implement `PrintableRequest[SearchDefinition]` (and indeed `Show[SearchDefinition]`), rather than having them be defined separately?

Reply:
I think maybe separate objects for now so each implementation isn't massive ?

Reply:
Although if I guess there's duplication now otherwise, so yeah maybe merge them in.